<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>optionsDialog</class>
 <widget class="QDialog" name="optionsDialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>436</width>
    <height>235</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Opciones</string>
  </property>
  <property name="windowIcon">
   <iconset resource="nxspooler.qrc">
    <normaloff>:/img/nxspooler.png</normaloff>:/img/nxspooler.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Opciones de NxSpooler</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="m_seconds_label">
        <property name="text">
         <string>&amp;Intervalo:</string>
        </property>
        <property name="buddy">
         <cstring>m_seconds</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="m_seconds">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>0</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Intervalo de tiempo en segundos para la búsqueda de documentos.</string>
        </property>
        <property name="whatsThis">
         <string>Intervalo de tiempo en segundos para la búsqueda de documentos.</string>
        </property>
        <property name="suffix">
         <string> s</string>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="value">
         <number>3</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="m_app_label">
        <property name="text">
         <string>&amp;Aplicación PDF:</string>
        </property>
        <property name="buddy">
         <cstring>m_app</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="m_app">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>230</width>
          <height>0</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Aplicación lectora de archivos PDF</string>
        </property>
        <property name="whatsThis">
         <string>Aplicación a utilizar para abrir los documentos PDF. No suele ser necesario cambiar este ajuste, ya que, dependiendo del sistema, se establece la mejor opción automáticamente.</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="m_path_label">
        <property name="text">
         <string>&amp;Ruta:</string>
        </property>
        <property name="buddy">
         <cstring>m_path</cstring>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="m_path">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>230</width>
          <height>0</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Ruta compartida</string>
        </property>
        <property name="whatsThis">
         <string>Ruta compartida mediante el nombre de recurso indicado debajo.</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="m_shared_label">
        <property name="text">
         <string>Recurso &amp;compartido:</string>
        </property>
        <property name="buddy">
         <cstring>m_shared</cstring>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLineEdit" name="m_shared">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>230</width>
          <height>0</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Nombre del recurso compartido en red</string>
        </property>
        <property name="whatsThis">
         <string>Nombre del recurso compartido en red  que el cliente NX montará en el servidor. NxSpooler no comprueba la existencia del recurso compartido, ya que no es necesario que el usuario actual tenga permisos para acceder a él.</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QToolButton" name="m_find_app">
        <property name="toolTip">
         <string>Buscar aplicación</string>
        </property>
        <property name="whatsThis">
         <string>Explorar el disco en busca de una aplicación.</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QToolButton" name="m_find_path">
        <property name="toolTip">
         <string>Buscar directorio</string>
        </property>
        <property name="whatsThis">
         <string>Explorar el disco en busca del directorio compartido.</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="spacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::MinimumExpanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="m_buttonBox">
     <property name="whatsThis">
      <string/>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok|QDialogButtonBox::RestoreDefaults</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>m_app</tabstop>
  <tabstop>m_find_app</tabstop>
  <tabstop>m_path</tabstop>
  <tabstop>m_find_path</tabstop>
  <tabstop>m_shared</tabstop>
  <tabstop>m_buttonBox</tabstop>
  <tabstop>m_seconds</tabstop>
 </tabstops>
 <resources>
  <include location="nxspooler.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>m_buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>optionsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>257</x>
     <y>225</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>optionsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>325</x>
     <y>225</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
